packages/pilot/src/components/CreditCardRefundDetails/index.js

```diff
@@ -73,21 +76,82 @@ const CreditCardRefundDetails = ({ contents, labels, title }) => {
             <Col palm={12} tablet={5} desk={5} tv={5}>
               {data.installments}
             </Col>
-
-            <Col palm={12} tablet={7} desk={7} tv={7}>
+          </Row>
+          <Row>
+            <Col
+              palm={12}
+              tablet={
+                recipient
+                  ? 4
+                  : 7
+              }
+              desk={
+                recipient
+                  ? 4
+                  : 7
+              }
+              tv={
+                recipient
+                  ? 4
+                  : 7
+              }
```

Comment:
aqui poderia setar uma variavel pro valor uma vez só ao inves de fazer o ternario 3x, o que acha?
```suggestion
const colSize = recipient  ? 4 : 7
....
               tablet={colSize}
              desk={colSize}
              tv={colSize}
```